### PR TITLE
Add CSS Handles to CostumerInfo's component

### PR DESCRIPTION
**Pull request by LucasCastroAcctGlobal**
#### What is the purpose of this pull request?

Add CSS Handles for the order-details component.

#### What problem is this solving?

Adds CSS classes so that you have greater flexibility in using items rendered by the order-details component.

### How should this be manually tested?
- Enter this link: https://orderplaced--tackoftheday.myvtex.com/checkout/orderPlaced/?og=1018171103781
- Inspect the element for the order-details component information;
- You will see that the new CSS Handles have been added.

### Screenshots or example usage
https://cl.ly/9022af914444

Types of changes
- [ ] - Bug fix (a non-breaking change which fixes an issue)
- [X] - New feature (a non-breaking change which adds functionality)
- [ ] - Breaking change (fix or feature that would cause existing functionality to change)
- [ ] - Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -298,3 +298,15 @@ interface Props {
...
+
+In order to apply CSS customizations in this and other blocks, follow the instructions given in the recipe on [Using CSS Handles for store customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization).
+
+| CSS Handles      |
+| ---------------- |
+| `costumerInfoListContainer`   |
+| `costumerInfoListName`   |
+| `costumerInfoListEmail`   |
+| `costumerInfoListDocument`   |
+| `costumerInfoListPhone`   |
```
Comment (klzns) [inline docs/README.md]:
```suggestion
| `customerInfoListContainer`   |
| `customerInfoListName`   |
| `customerInfoListEmail`   |
| `customerInfoListDocument`   |
| `customerInfoListPhone`   |
```

File: react/CustomerInfo.tsx [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
 import { ProfileRules, ProfileSummary } from 'vtex.profile-form'
+import { useCssHandles } from 'vtex.css-handles'
+
+const CSS_HANDLES = [
+  'costumerInfoListContainer',
+  'costumerInfoListName',
+  'costumerInfoListEmail',
+  'costumerInfoListDocument',
+  'costumerInfoListPhone'
+]
```
Comment (klzns) [inline react/CustomerInfo.tsx]:
```suggestion
] as const
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- CSS Handles for CostumerInfo component
```
Comment (klzns) [inline CHANGELOG.md]:
```suggestion
- CSS Handles for CustomerInfo component
```

**Review by klzns — CHANGES_REQUESTED** [review]
Thanks for the PR! There's a typo in customer

**Review by klzns — APPROVED** [review]

**Review by salesfelipe — APPROVED** [review]

**Review by bivillar — APPROVED** [review]

**Comment by klzns** [comment]
@allcontributors add @LucasCastroAcctGlobal for code

**Pull request merged**